service/deployer/eventer/eventer.go

```diff
@@ -0,0 +1,99 @@
+package eventer
+
+import (
+	"time"
+
+	microerror "github.com/giantswarm/microkit/error"
+	micrologger "github.com/giantswarm/microkit/logger"
+)
+
+// EventerType represents the type of Eventer to configure.
+type EventerType string
+
+// Config represents the configuration used to create an Eventer.
+type Config struct {
+	// Dependencies.
+	Logger micrologger.Logger
+
+	// Settings.
+	Type EventerType
+}
+
+// DefaultConfig provides a default configuration to create a new Eventer
+// service by best effort.
+func DefaultConfig() Config {
+	return Config{
+		// Dependencies.
+		Logger: nil,
+
+		// Settings.
+		Type: StubEventer,
+	}
+}
+
+// New creates a new configured Eventer.
+func New(config Config) (Eventer, error) {
+	// Dependencies.
+	if config.Logger == nil {
+		return nil, microerror.MaskAnyf(invalidConfigError, "logger must not be empty")
+	}
+
+	var newService Eventer
+
+	switch config.Type {
+	case StubEventer:
+		newService = &stubEventer{
+			// Dependencies.
+			logger: config.Logger,
+		}
+	default:
+		return nil, microerror.MaskAnyf(invalidConfigError, "could not find eventer type")
+	}
+
+	return newService, nil
+}
+
+// StubEventer is an Eventer that just pretends it's a real Eventer.
+var StubEventer EventerType = "StubEventer"
+
+// stubEventer is a stub implementation of the Eventer interface.
+type stubEventer struct {
+	// Dependencies.
+	logger micrologger.Logger
+}
+
+func (e *stubEventer) NewDeploymentEvents() (chan DeploymentEvent, error) {
+	e.logger.Log("debug", "watching for deployment events")
+
+	ticker := time.NewTicker(10 * time.Second)
```

Comment:
This is what I meant. This should be configurable. I am fine with defining the default in here. Having a constant and godoc for it would also be nice. Lets try to make everything injectable and documented by default. Our lives will be way easier in the future. When you do not want to provide a flag at the moment, fine, but I would like to see the service being configurable. This will also pay back when we test our code. When we talk about testing. Are there tests? Would it make sense to test the mechanism? How is the deployment events channel being canceled? Do we need to be able to cancel it? Please create issues where it makes sense.